### PR TITLE
feat: "Ethereum grants" page created #2479

### PR DESCRIPTION
**Pull request by ipapandinas**
## Description

Creation of a `Grants` subpage related to the [Community page](https://ethereum.org/en/community/).

Tell me what you think of the content @samajammin ? Is this what you had in mind ?

Imo it is a good idea to separate it on a subpage because the Community's main page deals more with topics related to the social aspect.

## Related Issue

#2479

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -225,8 +225,8 @@ const Footer = () => {
           text: "ef-blog",
         },
         {
-          to: "https://esp.ethereum.foundation",
-          text: "esp",
+          to: "/community/grants",
```
Comment (samajammin) [inline src/components/Footer.js:228]: I think we still want to link out to esp.ethereum.foundation. Let's add this as an additional link vs. replacing this one.

**Review by samajammin — COMMENTED** [review]
I think this is a great start! Thanks for the PR @ipapandinas ❤️ 

@ryancreatescopy mind taking a look at this & let us know what you think? I think we could add some additional info on the Grants page to make it more useful.

**Comment by ryancreatescopy** [comment]
Agree that this is a great start. I'd like if possible to categorise the grants programs. As someone looking for a grant right now I'd have to click on every link to find out which is most suitable. 

Do they offer different types of grants that we can categorise by?
- some offering research grants vs bug-finding vs product incubation etc?
Do some only offer up to certain amounts?
Are some community/DAO-based as opposed to less transparent granting models?

Separating quadratic funding could be good too as it's a very different / new type of grant model. We're planning a page on this eventually anyway, so would be good to separate it. 

I understand @ipapandinas if you don't have this information or the time to source, but this is the direction I'd want to take this in. 

Ultimately the goal of the page: help users find the right grant for them rather than create a space for grants to advertise their programs.

**Comment by samajammin** [comment]
Great idea @ryancreatescopy!

@ipapandinas would you be interested in making these improvements? Please let us know. Thanks again!

**Review by samajammin — CHANGES_REQUESTED** [review]
See comments above 😄

**Review by wackerow — APPROVED** [review]
@ipapandinas Thanks for getting this started! Hope you don't mind that I jumped in to try and get this moving along. Really appreciate you putting this list together though.
Made some changes as suggested by @ryancreatescopy above by breaking these into categories. Based it on the info available on each of their websites. A couple links were well past their deadlines so I removed them, and the Unicef link has a broken application link and I withheld it. 

@ryancreatescopy Feel free to drop suggestions. Kept the QF portion simple for now.

**Comment by ipapandinas** [comment]
Thanks for your review @ryancreatescopy !
Good job @wackerow 😃

I just added these changes:
- Regarding the Consensys grant program, the given grants are 10K$ or 25K$. However this program seems outdated, their Wave III submission application deadline was January 14, 2020. We should removed the link.
- MetaCartel has a dedicated grants page, we should use this link instead: https://www.metacartel.org/grants
- Grants ranging are provided for MetaCartel and Maker

File: src/content/community/grants/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+title: Ethereum Foundation & community grant programs
+description: A listing of the grant programs throughout the Ethereum ecosystem.
+lang: en
+---
+
+# Ethereum Grants {#ethereum-grants}
+
+The programs listed below offer a variety of funding grants for projects working to promote the success and growth of the Ethereum ecosystem. Use this as a guide to find and apply for funds to help make your next Ethereum project a success.
+
+### Broader Ecosystem {#broader-ecosystem}
```
Comment (ryancreatescopy) [inline src/content/community/grants/index.md:11]: broader than what?

File: src/content/community/grants/index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- [EF Ecosystem Support Program](https://esp.ethereum.foundation) - _Rollups, community and education, tooling and testing, infrastructure, interoperability, standards and research, wallets, and more_
+- [Consensys](https://consensys.net/grants/) - _Infrastructure L1 and L2, education & technical knowledge, usability and dev tooling, social impact, security, and more_
+- [MetaCartel](https://www.metacartel.org/) - _Dapp development_
+- [Moloch DAO](https://www.molochdao.com/) - _Privacy, layer 2 scaling, client security, and more_
+- [Open Grants](https://opengrants.com/explore)
+
+### Project Oriented {#project-oriented}
+
```
Comment (ryancreatescopy) [inline src/content/community/grants/index.md:20]: Can we add some explanation under the title. "These projects have their own grants for developing and experimenting with their technology." Or something like that. Project-oriented isn't immediately clear to me.

File: src/content/developers/docs/intro-to-ethereum/index.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ Watch Austin walk you through blockchains:
 
 <iframe width="100%" height="315" src="https://www.youtube.com/embed/zcX7OJ-L8XQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-If you want to see how blockchain hashes data, and then the previous block references all the past blocks, be sure to check out [this demo](https://andersbrownworth.com/blockchain/blockchain). It makes understanding the hashing of blocks really simple. 
+If you want to see how blockchain hashes data, and then the previous block references all the past blocks, be sure to check out [this demo](https://andersbrownworth.com/blockchain/blockchain). It makes understanding the hashing of blocks really simple.
```
Comment (ryancreatescopy) [inline src/content/developers/docs/intro-to-ethereum/index.md:28]: This is a confusing sentence. Maybe something like: "If you want to see how the blockchain hashes data, and how a block references all past blocks, be sure to check out [this demo]. It makes understanding the hashing of blocks really simple."

File: src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md [hunk 5]
```diff
@@ -145,6 +145,6 @@ The latest block number is 11043912
 
 ‌**Woo! Congrats! You just wrote your first web3 script using Alchemy 🎉**
 
-‌Not sure what to do next? Try deploying your first smart contract and get your hands dirty with some solidity programming in our [_Hello World Smart Contract Guide](https://docs.alchemyapi.io/tutorials/hello-world-smart-contract), or test your dashboard knowledge with the [_Dashboard Demo App_](https://docs.alchemyapi.io/tutorials/demo-app)!
+‌Not sure what to do next? Try deploying your first smart contract and get your hands dirty with some solidity programming in our [\_Hello World Smart Contract Guide](https://docs.alchemyapi.io/tutorials/hello-world-smart-contract), or test your dashboard knowledge with the [_Dashboard Demo App_](https://docs.alchemyapi.io/tutorials/demo-app)!
```
Comment (ryancreatescopy) [inline src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md:148]: ?

File: src/content/community/grants/index.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+- [Maker](https://grants.makerdao.com/) - _Risk, governance, core development, ecosystem, tools, MarerDAO community, security_
+- [TheGraph](https://airtable.com/shreX09LazIhsg0bU)
+- [Uniswap](https://airtable.com/shrEXXxXB1humz7VS)
+
+### Quadratic Funding {#quadratic-funding}
+
```
Comment (ryancreatescopy) [inline src/content/community/grants/index.md:27]: I think we need at least a one-line explainer here. Take something from the [DeFi](https://ethereum.org/en/defi/#quadratic-funding) explanation of it.

**Review by ryancreatescopy — CHANGES_REQUESTED** [review]
There's a lot of random stuff in this PR like removal and addition of spaces. Not sure what's going on, but went through the diff and made some comments. The grants page is really close

File: src/content/developers/docs/intro-to-ethereum/index.md (hunk 4 above)
Comment (wackerow) [reply src/content/developers/docs/intro-to-ethereum/index.md:28]: this looks like an automatic linting change that removed a trailing space. similar below, unrelated to this PR

File: src/content/community/grants/index.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+---
+title: Ethereum Foundation & community grant programs
+description: A listing of the grant programs throughout the Ethereum ecosystem.
```
Comment (samajammin) [inline src/content/community/grants/index.md:3]:
```suggestion
description: A listing of the grant programs throughout the Ethereum ecosystem.
sidebar: true
```

**Review by samajammin — APPROVED** [review]
LGTM for a v1! @ryancreatescopy could you please review the latest changes?

**Pull request merged**